src/hooks/sdk/useAttendeeVideoMetrics.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useState, useEffect } from 'react';
+import { ClientMetricReport } from 'amazon-chime-sdk-js';
+
+import { useAudioVideo } from '../../providers/AudioVideoProvider';
+
+interface AttendeeVideoMetrics { 
+  streams: { [id: string]: {[key: string]: number} };
```

Comment:
Should we change this `id` to be `attendeeId` just to clarify what the mapping is?

Reply:
Do we need to change it? This is only for defining typescript index signature. We are not gonna use it as variable.

Reply:
If you want you can not required as such, I just thought it would help us understand that it is attendeeId -> value mapping. But, fine if we don't do it.

Reply:
Gotcha. I will change the index here to make it more readable. Thanks for pointing this out.